experimenter/experimenter/experiments/api/v8/serializers.py

```diff
@@ -179,6 +181,9 @@ def get_outcomes(self, obj):
             for slug in outcomes
         ]
 
+    def get_segments(self, obj):
+        return [{"slug": slug} for slug in obj.segments]
```

Comment:
I was going to say we don't need to wrap it in a `{'slug': ....}` object since it's already only a slug, but then I noticed it's just mirroring the same pattern/interface for outcomes above, and theoretically this lets us add additional keys if we need to later so actually probably just fine to leave it like this.